### PR TITLE
[WIP] New Hamburg neighbourhoods and boroughs

### PR DESCRIPTION
**Pull request by stepps00**
WIP. Will fix https://github.com/whosonfirst-data/whosonfirst-data/issues/1816

This PR:

- [x] Adds seven new borough records representing [the boroughs of Hamburg](https://en.wikipedia.org/wiki/Boroughs_and_quarters_of_Hamburg#Borough_overview). These records have concordances, names, and source properties when available from superseding existing records.
- [x] Adds polygons to 100+ existing neighbourhood records, representing [the quarters of Hamburg](https://en.wikipedia.org/wiki/Boroughs_and_quarters_of_Hamburg#Quarters). 
- [x] Flags "leftover" neighbourhood and locality records as is_funky=1.

Once approved, I'll PIP this PR to update hierarchies and validate the geojson files/geometries.

### CODE REVIEW

**Comment by stepps00** [comment]
New neighbourhood geometries:
Screenshot 2023-04-07 at 3 18 45 PM

New borough geometries:
Screenshot 2023-04-07 at 3 19 22 PM

**Comment by stepps00** [comment]
@nvkelso this PR is ready for review.. I'll PIP and validate once the changes get a :+1:

File: data/184/752/401/5/1847524015.geojson [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    "geom:latitude":53.617616,
+    "geom:longitude":10.016526,
+    "iso:country":"DE",
+    "lbl:latitude":53.610715,
+    "lbl:longitude":10.01212,
+    "mps:latitude":53.610715,
+    "mps:longitude":10.01212,
+    "mz:hierarchy_label":1,
+    "mz:is_current":1,
+    "name:deu_x_preferred":[
```
Comment (nvkelso) [inline data/184/752/401/5/1847524015.geojson:31]: For DEU and ENG `Nord` should be preferred name, and `Hamburg-Nord` a variant (and possibly a long name for disambiguation)

File: data/184/752/401/9/1847524019.geojson [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    "geom:latitude":53.54225,
+    "geom:longitude":9.948146,
+    "iso:country":"DE",
+    "lbl:latitude":53.51805,
+    "lbl:longitude":10.013092,
+    "mps:latitude":53.51805,
+    "mps:longitude":10.013092,
+    "mz:hierarchy_label":1,
+    "mz:is_current":1,
+    "name:deu_x_preferred":[
```
Comment (nvkelso) [inline data/184/752/401/9/1847524019.geojson:34]: For DEU and ENG `Mitte` should be preferred name, and `Hamburg-Mitte` a variant (and possibly a long name for disambiguation). Also consider modifying wof:name.

**Comment by nvkelso** [comment]
I wonder if the neighbourhoods marked `is_funky` in https://github.com/whosonfirst-data/whosonfirst-data-admin-de/pull/88/commits/cf92f43643ec800128dac7503b1412ed19da3759 and https://github.com/whosonfirst-data/whosonfirst-data-admin-de/pull/88/commits/a86dcd4857d097960dab149a683deccb1cb56914 are actually microhoods? The polygons would certainly be wrong... but consider investigating for 10 minutes if their placetype should be recast and min_zooms adjusted down?

**Comment by stepps00** [comment]
[b557af2](https://github.com/whosonfirst-data/whosonfirst-data-admin-de/pull/88/commits/b557af2e86c4ff02eea07f27e3a81b5c319746e9) moves the "funky" neighbourhoods to the microhood placetype, I also adjusted the names on two of the borough records.

@nvkelso if this looks good, I'll PIP the branch and merge :+1:

**Review by nvkelso — APPROVED** [review]
LGTM, thanks!

**Comment by stepps00** [comment]
PIP work added in the last commit, merging this PR.

**Pull request merged**